src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1440,6 +1440,14 @@ class ApplicationManagerImpl
   static std::vector<std::string> ConvertRejectedParamList(
       const std::vector<std::string>& input);
 
+  /**
```

Comment:
@KVGrygoriev missed arg names in description
@param app_to_remove the application which should be removed
@param is_unexpected_disconnect specifies whether the disconnect was unexpected

Reply:
@EKuliiev thanks, fixed